### ISSUE TITLE
[3.2.2 beta3] Tween _build_interpolation: Initial value type 'Nil' does not match final value type 'Vector2'.

### ISSUE THREAD

**Issue by git2013vb** [godotengine/godot#38988]
**Godot version:**
3.2.2 beta3 mono
**OS/device including version:**
Debian 10
**Issue description:**
I have this error when I set `initVal` to `null` using `InterpolateProperty` in Tween
I did't have this error in 3.2.1.mono.stable
```
E 0:00:19.572   _build_interpolation: Initial value type 'Nil' does not match final value type 'Vector2'.
  <C++ Error>   Condition "p_initial_val.get_type() != p_final_val.get_type()" is true. Returned: false
  <C++ Source>  scene/animation/tween.cpp:1289 @ _build_interpolation()
  <Stack Trace> :0 @ Boolean Godot.NativeCalls.godot_icall_8_870(IntPtr , IntPtr , IntPtr , IntPtr , System.Object , System.Object , System.Single& , Int32 , Int32 , System.Single& )()
                Tween.cs:412 @ Boolean Godot.Tween.InterpolateProperty(Godot.Object , Godot.NodePath , System.Object , System.Object , Single , Godot.Tween+TransitionType , Godot.Tween+EaseType , Single )()
                Move2D.cs:166 @ void Move2D.TravelTo(Godot.Vector2 , UInt64 , Single )()
                Move2D.cs:97 @ void Move2D.ApplyInputs(Single )()
                Move2D.cs:53 @ void Move2D._PhysicsProcess(Single )()

```
**Steps to reproduce:**
see before
**Minimal reproduction project:**
NA

**Comment by KoBeWi** [comment]
This works in GDScript btw.

**Comment by neikeq** [comment]
I tried to reproduce this from what I understood from the issue, but I don't get any error so I may have done something wrong. Can you upload an example project to reproduce the issue?

**Comment by akien-mga** [comment]
> Can you upload an example project to reproduce the issue?

Bump. This likely won't be fixed in 3.2.3 if we can't reproduce it.

**Comment by akien-mga** [comment]
No answer, so closing. Please comment if you can still reproduce it, and make sure that an up-to-date reproduction project is included.
